appeals/web/src/client/components/add-another/add-another.js

```diff
@@ -110,6 +114,11 @@ function addAnotherItem(componentInstance) {
 
 	items = getItems(componentInstance);
 
+	if (items.length > 1) {
+		addRemoveButton(items[0]);
```

Comment:
Edit: sorry, I misread this as adding the remove button to the new item - I see what this is doing now

Reply:
Just a suggestion, but this might be clearer if it was moved into its own function (called something like `updateRemoveButtons`) and called from here